jsonapi/fixtures_test.go

```diff
@@ -272,12 +353,17 @@ func (c *Post) SetReferencedStructs(references []UnmarshalIdentifier) error {
 
 type AnotherPost struct {
 	ID       int   `json:"-"`
+	LID      int   `json:"-"`
 	AuthorID int   `json:"-"`
 	Author   *User `json:"-"`
 }
 
-func (p AnotherPost) GetID() string {
-	return fmt.Sprintf("%d", p.ID)
+func (p AnotherPost) GetID() Identifier {
+	id := Identifier{ID: fmt.Sprintf("%d", p.ID), Name: "another-posts"}
```

Comment:
No need to define Name, since GetName was not implemented before

Reply:
removed